test/services/abandon_solution_mentorship_test.rb

```diff
@@ -17,7 +17,7 @@ class AbandonSolutionMentorshipTest < ActiveSupport::TestCase
 
     # Check discussion post has been created
     content = {'i18n_message' => 'system_messages.mentor_timed_out', 'handle' => mentor.handle}
-    html = ParseMarkdown.("Mentor 'freddie' has being removed from this conversation due to inactivity. This solution will be placed into the queue for a new mentor to review.")
+    html = ParseMarkdown.("Mentor 'freddie' been being removed from this conversation due to inactivity. This solution has been resubmitted to the queue for a new mentor to review.")
```

Comment:
s/been being/had been

Reply:
```suggestion
    html = ParseMarkdown.("Mentor 'freddie' has been removed from this conversation due to inactivity. This solution has been resubmitted to the queue for a new mentor to review.")
```

Reply:
@BethanyG @F3PiX Thanks both. It's taken me three attempts to do this! More coffee needed!

Reply:
🥳